docs/how-to-work-on-coding-challenges.md

```diff
@@ -209,13 +209,15 @@ Our goal is to have thousands of 2-minute challenges. These can flow together an
 Here are specific formatting guidelines for challenge text and examples:
 
 - Language keywords go in `<code>` tags. For example, HTML tag names or CSS property names
-- The first instance of a keyword when it's being defined, or general keywords (i.e. "object" or "immutable") go in `<dfn>` tags
+- The first instance of a keyword when it's being defined, or general keywords (e.g. "object" or "immutable") go in `<dfn>` tags
 - References to code parts (i.e. function, method or variable names) should be wrapped in `<code>` tags. See example below:
-- Use <code>parseInt</code> to convert the variable <code>realNumber</code> into an integer.
-- Multi-line code blocks **must be preceded by an empty line**. The next line must start with three backticks followed immediately by one of the [supported languages](https://prismjs.com/#supported-languages). To complete the code block, you must start a newline which only has three backticks and **another empty line**.
-  **Note:** If you are going to use an example code in YAML, use `yaml` instead of `yml` for the language to the right of the backticks.
-
-See example below:
+```md
+Use <code>parseInt</code> to convert the variable <code>realNumber</code> into an integer.
+```
+- References to file names and path directories (e.g. `package.json`, `src/components`) should be wrapped in `<code>` tags.
+- Multi-line code blocks **must be preceded by an empty line**. The next line must start with three backticks followed immediately by one of the [supported languages](https://prismjs.com/#supported-languages). To complete the code block, you must start a newline which only has three backticks and **another empty line**. See example below:
+  
+**Note:** If you are going to use an example code in YAML, use `yaml` instead of `yml` for the language to the right of the backticks.
```

Comment:
```suggestion
**Note:** If you are going to use an example code in YAML, use `yaml` instead of `yml` for the language, to the right of the backticks.
```

Reply:
I would be happy to accept this, but, if I wrote the NOTE, I would have put it:
> If you are going to use an example code in YAML, use `yaml` instead of `yml`, for the language to the right of the backticks.

Would you agree, @Manish-Giri ?

Reply:
@Manish-Giri I agree with @Sky020 with respect to not including the comma.  It seems to unnecessarily break up the flow of the sentence.

Reply:
okay.